tests/Oryx.BuildImage.Tests/Python/PythonSampleAppsTest.cs

```diff
@@ -946,7 +946,7 @@ public void BuildPythonApps_AndHasLzmaModule(string appName)
             var appOutputDir = "/tmp/app1-output";
             var script = new ShellScriptBuilder()
                 .AddDefaultTestEnvironmentVariables()
-                .AddBuildCommand($"{appDir} -o {appOutputDir} --platform python --platform-version {PythonVersions.Python310Version}")
+                .AddBuildCommand($"{appDir} -o {appOutputDir} --platform python --platform-version 3.10.8")
```

Comment:
Think we can keep this change to use {PythonVersions.Python310Version} as-is since we are rolling back to version 3.10.8 anyway.

Reply:
Updated